docs/README.md

```diff
@@ -279,6 +279,18 @@ const isCi = process.env.CI !== undefined
 if (!isCi) {
   require('husky').install()
 }
+
+
+You can also use your own logging functions if needed:
```

Comment:
Should this be outside if the code formatted block?

Reply:
You're right, I messed up while resolving conflicts.
I'll fix that right away.

I miss so much the unit tests for documentation ;-).

Reply:
fixed.

Reply:
LGTM